packages/genesys-spark-components/src/components/stable/gux-rating/example.html

```diff
@@ -8,11 +8,19 @@ <h2>Defaults</h2>
 >
 </gux-rating>
 
+<h2>Integers only</h2>
+<gux-rating
+  aria-label="feedback"
+  increment="1"
+  onchange="notify(event)"
+  oninput="notify(event)"
+>
+</gux-rating>
+
 <h2>Set initial value</h2>
-<label for="rating">Feedback</label><br />
+<label id="rating">Feedback</label><br />
```

Comment:
Do we need an `id` here if it's not used for anything?

Reply:
Used by the `aria-labelledby` below for accessibility.